tests/cache_test.py

```diff
@@ -172,3 +172,12 @@ def lambda_handler(event, context):
             return 'Hello from Lambda with secret %s' % secret_value
         
         lambda_handler(None, None)
+
+    @patch('ssm_cache.cache.SSMParameterStore.SESSION_FACTORY', new=None)
```

Comment:
do we really need the full path here?

it can't just be `@patch('SSMParameterStore.SESSION_FACTORY', new=None)`?

Reply:
it didn't work for me without; I'll try again

Reply:
confirmed, it still didn't work for me without the full path.